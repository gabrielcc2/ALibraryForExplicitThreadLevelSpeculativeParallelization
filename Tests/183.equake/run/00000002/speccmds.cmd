-u /home/gabriel/Escritorio/SPEC_CPU2000_CD_2/benchspec/CFP2000/183.equake/run/00000002
-i inp.in >inp.out 2>inp.err ../00000002/equake_base.x86_linux
