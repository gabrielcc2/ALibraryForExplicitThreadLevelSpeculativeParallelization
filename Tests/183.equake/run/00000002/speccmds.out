running commands in speccmds.cmd 1 times
runs started at 1334880529, 220066000, Thu Apr 19 19:38:49 2012
run 1 started at 1334880529, 220167000, Thu Apr 19 19:38:49 2012
child started: 0, 1334880529, 220180000, 2750, '>inp.out 2>inp.err ../00000002/equake_base.x86_linux'
child finished: 0, 1334880529, 712236000, 0, 492056000, 2750, 0
run 1 finished at: 1334880529, 712243000, Thu Apr 19 19:38:49 2012
run 1 elapsed time: 0, 492076000, 0.492076000
runs finished at 1334880529, 712287000, Thu Apr 19 19:38:49 2012
runs elapsed time: 0, 492221000, 0.492221000
